android-app-with-rn/android/app/build.gradle

```diff
@@ -63,6 +66,15 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+
+    constraints {
```

Comment:
what causes the need to add this? normally if all libs are resolved to `1.8`, there shouldn't be a need for that.

Also it is maybe worth to use the latest patch release for `1.8` (`1.8.22`, I guess).

Reply:
I had the usual Kotlin compiling issue when not adding this bit.
Can we solve this in a different way?